packages/flutter/lib/src/material/expansion_tile.dart

```diff
@@ -491,6 +495,39 @@ class ExpansionTile extends StatefulWidget {
   /// than supplying a controller.
   final ExpansionTileController? controller;
 
+  /// Whether this list tile is part of a vertically dense list.
```

Comment:
Looks like this documentation is the same as `ListTile.dense`. Could you help put a `{@template ...}` over there and use `{@macro}` here?

Reply:
> Looks like this documentation is the same as `ListTile.dense`. Could you help put a `{@template ...}` over there and use `{@macro}` here?

There is no template for `ListTile.dense`. Should change its doc in `list_tile.dart`?

Reply:
Yes, you can use `{@template } {@endtemplate}` to wrap the doc of `ListTile.dense`, and use {@macro} for `dense` in `ExpansionTile`. Here's the instruction: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#dartdoc-templates-and-macros